### PR TITLE
First pass through updating

### PR DESCRIPTION
**Pull request by barrettclark**
Mostly removed some stuff. Next pass will fine-tune and add more stuff.

### CODE REVIEW

**Comment by barrettclark** [comment]
Can haz review @petesharum

File: STYLEGUIDE.md [hunk 1]
```diff
@@ -188,12 +188,22 @@ STATES = ["draft", "open", "closed"]
 STATES = %w(draft open closed)
 ```
 
+* Prefer `%i` to the literal array syntax when you need an array of symbols (and you don't need to maintain Ruby 1.9 compatibility). Apply this rule only to arrays with two or more elements.
```
Comment (petesharum) [inline STYLEGUIDE.md]: Suggestion: remove sentence stating "Apply this rule only to arrays with two or more elements." If we prefer a particular syntax for array literals why maintain a special case for single-element arrays?

File: STYLEGUIDE.md [hunk 2]
```diff
@@ -635,9 +612,9 @@ names.select do |name|
 end.map { |name| name.upcase }
 ```
 
-    Some will argue that multiline chaining would look OK with the use of {...}, but they should
-    ask themselves - is this code really readable and can't the block's contents be extracted into
-    nifty methods?
+Some will argue that multiline chaining would look OK with the use of {...}, but they should
+ask themselves - is this code really readable and can't the block's contents be extracted into
+nifty methods? Don't just write clever or cute code.
```
Comment (petesharum) [inline STYLEGUIDE.md]: Write clever _and_ cute code. ;-)

File: STYLEGUIDE.md [hunk 3]
```diff
@@ -203,30 +213,9 @@ hash = { "one" => 1, "two" => 2, "three" => 3 }
...
-# Returns the duplicated String.
-def multiplex(text, count)
-  text * count
-end
-```
-
 ## Exceptions
 
-* Don't use exceptions for flow of control.
+* Don't use exceptions for flow control.
```
Comment (petesharum) [inline STYLEGUIDE.md]: According to [wikipedia](https://en.wikipedia.org/wiki/Control_flow), the correct term is either 'flow of control' or 'control flow'. We use 'flow of control' later in the document, so I suggest switching to that for consistency.

File: STYLEGUIDE.md [hunk 4]
```diff
@@ -351,14 +342,14 @@ remove_member user, skip_membership_check: true
...
-* Use `%w` freely.
+* Use `%w` and `%i` freely for arrays of strings and symbols.
 
 ``` ruby
 STATES = %w(draft open closed)
 ```
 
 * Use `%()` for single-line strings which require both interpolation
-  and embedded double-quotes. For multi-line strings, prefer heredocs.
+  and embedded double-quotes. For multi-line strings, use heredocs.
```
Comment (petesharum) [inline STYLEGUIDE.md:352]: Is "For multi-line strings, use heredocs." redundant with line 465?

**Review by petesharum — COMMENTED** [review]
I'm starting to review the rubocop config to update with our changes and found a couple of minor suggestions.

**Pull request merged**